spk/davfs2/Makefile

```diff
@@ -0,0 +1,29 @@
+SPK_NAME = davfs2
+SPK_VERS = 1.5.2
+SPK_REV = 3
+
+#icon downloaded from https://www.iconfinder.com/icons/32041/folder_web_icon#size=128
+#Free usage see http://mazenl77.deviantart.com/art/NX11-Unfinished-107836821
```

Comment:
Explanation is appreciated, but keep it a one-liner if you need it at all. Just the download link would be sufficient imo, because it already states the license limitations or links to it.